bindings/nodejs/lib/index.ts

```diff
@@ -39,3 +39,48 @@ export * from './types';
 export * from './utils';
 export * from './wallet';
 export * from './logger';
+
+// For future reference to see what we return from rust as a serialized string
+export type Result = {
+    // "error" | "panic", or other binding "Response" enum name, we consider "ok".
+    type: string;
+    // "panic" means payload is just a string, otherwise its the object below.
+    payload: {
+        // Ok: All method names from types/bridge/__name__.name
+        // Not ok: all variants of iota_sdk_bindings_core::Error type i.e block/client/wallet/
+        type: string;
+        // If "ok", json payload
+        payload?: string;
+        // If not "ok", json error
+        error?: string;
+    };
+};
+
+function errorHandle(error: any): Error {
+    try {
+        const err: Result = JSON.parse(error.message);
+        if (!err.type) {
+            return error;
+        }
+
+        if (err.type == 'panic') {
+            // Panic example:
+            // {"type":"panic","payload":"Client was destroyed"}
+            return Error(err.payload.toString());
+        } else if (err.type == 'error') {
+            // Error example:
+            // {"type":"error","payload":{"type":"client","error":"no healthy node available"}}
+            // TODO: switch on type and create proper js errors
```

Comment:
wen? Or do we have an issue?

Reply:
there is one made a long time ago, firefly requested it. let me see if i can find it

Reply:
Maybe put it in the code then, it does help

Reply:
Found it; https://github.com/iotaledger/iota-sdk/issues/1417